packages/@aws-cdk/aws-ec2/lib/vpc-endpoint-service.ts

```diff
@@ -0,0 +1,87 @@
+import { ArnPrincipal } from '@aws-cdk/aws-iam';
+import { Construct, IResource, Resource } from '@aws-cdk/core';
+import { CfnVPCEndpointService, CfnVPCEndpointServicePermissions } from './ec2.generated';
+
+/**
+ * A VPC endpoint service.
+ */
+export interface IVpcEndpointService extends IResource {
```

Comment:
An empty interface seems odd? I would expect any EndpointService to have a serviceName at the very least, which means that should be part of the interface.

Reply:
Added serviceName